services/git-service-impl/src/main/java/io/fabric8/launcher/service/git/AbstractGitService.java

```diff
@@ -32,14 +36,39 @@ protected AbstractGitService(final Identity identity) {
         this.identity = identity;
     }
 
-    private static final Logger LOG = Logger.getLogger(AbstractGitService.class.getName());
+    private static final Logger logger = Logger.getLogger(AbstractGitService.class.getName());
 
     private static final String AUTHOR = EnvironmentSupport.INSTANCE.getEnvVarOrSysProp("LAUNCHER_MISSION_CONTROL_COMMITTER_AUTHOR", "openshiftio-launchpad");
 
     private static final String AUTHOR_EMAIL = EnvironmentSupport.INSTANCE.getEnvVarOrSysProp("LAUNCHER_MISSION_CONTROL_COMMITTER_AUTHOR_EMAIL", "obsidian-leadership@redhat.com");
 
     private final Identity identity;
 
+    @Override
+    public void clone(GitRepository repository, Path path) {
+        requireNonNull(repository, "repository must not be null.");
+        requireNonNull(path, "path must not be null.");
+        // Not using JGit here because it doesn't support shallow clones yet
+        ProcessBuilder builder = new ProcessBuilder()
+                .command("git", "clone", repository.getGitCloneUri().toString(),
```

Comment:
@ia3andy for some reason, Hoverfly doesn't work with native commands, can you help me figure out why?